src/components/molecules/SearchTermListItem/search-term-list-item.jsx

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import SynonymList from '../../atomic/SynonymList';
+
+import { Link } from 'react-router-dom';
+
+import './search-term-list-item.scss';
```

Comment:
Reference to the scss partial should be in the dictionaries.scss.  Now I'm seeing that this hasn't been done for other views and components.  Can we make some adjustments here?